applications/openshift/logging/file_ownership_var_log_kube_audit/rule.yml

```diff
@@ -0,0 +1,47 @@
+documentation_complete: true
+
+prodtype: ocp4
+
+title: 'Kubernetes Audit Logs Must Be Owned By Root'
+
+platforms:
+  - ocp4-master-node
+
+description: |-
+    All audit logs must be owned by root user and group. By default, the path for the Kubernetes audit log is <pre>/var/log/kube-apiserver/</pre>.
+    {{{ describe_file_owner(file="/var/log/kube-apiserver", owner="root") }}}
+    {{{ describe_file_owner(file="/var/log/kube-apiserver/*", owner="root") }}}
+
+rationale: |-
+    Unauthorized disclosure of audit records can reveal system and configuration data to
+    attackers, thus compromising its confidentiality.
+
+severity: medium
+
+identifiers:
+    cce@ocp4: CCE-83650-2
+
+references:
+    cjis: 5.4.1.1
+    cui: 3.3.1
+    disa: CCI-000162,CCI-000163,CCI-000164,CCI-001314
+    nist: CM-6(a),AC-6(1),AU-9(4)
+    nist-csf: DE.AE-3,DE.AE-5,PR.AC-4,PR.DS-5,PR.PT-1,RS.AN-1,RS.AN-4
+    pcidss: Req-10.5.1
+    srg: SRG-OS-000057-GPOS-00027,SRG-OS-000058-GPOS-00028,SRG-OS-000059-GPOS-00029,SRG-OS-000206-GPOS-00084
+    isa-62443-2013: 'SR 2.1,SR 2.10,SR 2.11,SR 2.12,SR 2.8,SR 2.9,SR 5.2,SR 6.1'
+    isa-62443-2009: 4.2.3.10,4.3.3.3.9,4.3.3.5.8,4.3.3.7.3,4.3.4.4.7,4.3.4.5.6,4.3.4.5.7,4.3.4.5.8,4.4.2.1,4.4.2.2,4.4.2.4
+    cobit5: APO01.06,APO11.04,APO12.06,BAI03.05,BAI08.02,DSS02.02,DSS02.04,DSS02.07,DSS03.01,DSS05.04,DSS05.07,DSS06.02,MEA02.01
+    iso27001-2013: A.10.1.1,A.11.1.4,A.11.1.5,A.11.2.1,A.12.4.1,A.12.4.2,A.12.4.3,A.12.4.4,A.12.7.1,A.13.1.1,A.13.1.3,A.13.2.1,A.13.2.3,A.13.2.4,A.14.1.2,A.14.1.3,A.16.1.4,A.16.1.5,A.16.1.7,A.6.1.2,A.7.1.1,A.7.1.2,A.7.3.1,A.8.2.2,A.8.2.3,A.9.1.1,A.9.1.2,A.9.2.3,A.9.4.1,A.9.4.4,A.9.4.5
+    cis-csc: 1,11,12,13,14,15,16,18,19,3,4,5,6,7,8
+
+ocil: |-
+    {{{ describe_file_owner(file="/var/log/kube-apiserver", owner="root") }}}
+    {{{ describe_file_owner(file="/var/log/kube-apiserver/*", owner="root") }}}
+
+template:
+    name: file_owner
+    vars:
+        filepath: ^/var/log/kube-apiserver($|/.*$)
+        filepath_is_regex: 'true'
+        fileuid: '0'
```

Comment:
This rule is good, just a question about group ownership - we don't need it because we assert that there are no permissions for the group right?

Reply:
that's right